### PR TITLE
[169945] Raise if a status code, which must not have body, has body

### PR DESCRIPTION
**Pull request by aYukiSekiguchi**
https://acsmine.tok.access-company.com/redmine/issues/169945

### CODE REVIEW

File: test/core/web/conn_test.exs [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      status: status,
+      resp_headers: %{},
+      resp_cookies: %{},
+      resp_body: body,
+      before_send: [],
+      assigns: %{}
+    }
+  end
+
+  describe "validate/1" do
```
Comment (Double-oxygeN) [inline test/core/web/conn_test.exs:21]: There is no test to check that a response with a status which must not have a body is valid when the body is actually empty.

**Review by Double-oxygeN — CHANGES_REQUESTED** [review]

File: test/core/web/conn_test.exs [hunk 2]
```diff
@@ -0,0 +1,37 @@
+# Copyright(c) 2015-2023 ACCESS CO., LTD. All rights reserved.
+
+defmodule AntikytheraCore.ConnTest do
+  use Croma.TestCase
+
+  alias Antikythera.Conn, as: LibConn
+
+  def make_fake_conn(status, body) do
```
[the remark this answers is not in the document]
Comment (aYukiSekiguchi) [reply test/core/web/conn_test.exs]: Thank you.
Fixed at https://github.com/access-company/antikythera/pull/234/commits/94f69c3576f8310460adfa70a8aaddb34be610d3

File: test/core/web/conn_test.exs (hunk 1 above)
Comment (aYukiSekiguchi) [reply test/core/web/conn_test.exs:21]: You're right.
Add test cases for empty body
https://github.com/access-company/antikythera/pull/234/commits/94f69c3576f8310460adfa70a8aaddb34be610d3

**Review by Double-oxygeN — APPROVED** [review]
LGTM
